modules/images/webp-uploads/can-load.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Can load function to determine if module already marge in WordPress core.
+ *
+ * @since   n.e.x.t
+ * @package performance-lab
+ */
+
+/**
+ * Returns an array with the list of valid mime types that a specific mime type can be converted into it,
+ * for example an image/jpeg can be converted into an image/webp.
+ *
+ * @since n.e.x.t
+ *
+ * @return bool True if the mofule module already marge in WordPress core, or false.
+ */
+
+function perflab_check_webp_uploads_core_functions() {
```

Comment:
And return the function

```suggestion
return function perflab_check_webp_uploads_core_functions() {
```

Reply:
We use `return 'perflab_check_webp_uploads_core_functions';` now it working fine. Thanks!